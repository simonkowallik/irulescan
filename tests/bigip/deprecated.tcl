accumulate
client_addr
client_port
decode_uri
findclass
http_cookie
http_header
http_host
http_method
http_uri
http_version
imid
ip_addr
ip_protocol
ip_tos
ip_ttl
link_qos
local_addr
local_port
matchclass
redirect
remote_addr
remote_port
server_addr
server_port
urlcatblindquery
urlcatquery
use
vlan_id