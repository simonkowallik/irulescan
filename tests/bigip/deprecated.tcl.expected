WARNING: Use of deprecated command: at `accumulate` in `accumulate`
WARNING: Use of deprecated command: at `client_addr` in `client_addr`
WARNING: Use of deprecated command: at `client_port` in `client_port`
WARNING: Use of deprecated command: at `decode_uri` in `decode_uri`
WARNING: Use of deprecated command: at `findclass` in `findclass`
WARNING: Use of deprecated command: at `http_cookie` in `http_cookie`
WARNING: Use of deprecated command: at `http_header` in `http_header`
WARNING: Use of deprecated command: at `http_host` in `http_host`
WARNING: Use of deprecated command: at `http_method` in `http_method`
WARNING: Use of deprecated command: at `http_uri` in `http_uri`
WARNING: Use of deprecated command: at `http_version` in `http_version`
WARNING: Use of deprecated command: at `imid` in `imid`
WARNING: Use of deprecated command: at `ip_addr` in `ip_addr`
WARNING: Use of deprecated command: at `ip_protocol` in `ip_protocol`
WARNING: Use of deprecated command: at `ip_tos` in `ip_tos`
WARNING: Use of deprecated command: at `ip_ttl` in `ip_ttl`
WARNING: Use of deprecated command: at `link_qos` in `link_qos`
WARNING: Use of deprecated command: at `local_addr` in `local_addr`
WARNING: Use of deprecated command: at `local_port` in `local_port`
WARNING: Use of deprecated command: at `matchclass` in `matchclass`
WARNING: Use of deprecated command: at `redirect` in `redirect`
WARNING: Use of deprecated command: at `remote_addr` in `remote_addr`
WARNING: Use of deprecated command: at `remote_port` in `remote_port`
WARNING: Use of deprecated command: at `server_addr` in `server_addr`
WARNING: Use of deprecated command: at `server_port` in `server_port`
WARNING: Use of deprecated command: at `urlcatblindquery` in `urlcatblindquery`
WARNING: Use of deprecated command: at `urlcatquery` in `urlcatquery`
WARNING: Use of deprecated command: at `use` in `use`
WARNING: Use of deprecated command: at `vlan_id` in `vlan_id`

